apps/src/templates/studioHomepages/ResourceCard.jsx

```diff
@@ -131,7 +145,12 @@ class ResourceCard extends Component {
 
     return (
       <div style={cardStyles}>
-        <div style={titleStyles}>{title}</div>
+        <div style={styles.titleContainer}>
+          <div style={titleStyles}>{title}</div>
+          <div style={styles.callout}>
```

Comment:
Do you still want to render this block if there's no callout?

Reply:
yeah i guess it's clearer if i make that conditional. i'll update when i remove that stray commented line after the build completes